haystack/document_stores/faiss.py

```diff
@@ -367,6 +369,8 @@ def update_embeddings(
                 if self.similarity == "cosine":
                     self.normalize_embedding(embeddings)
 
+                if not self.faiss_indexes[index].is_trained:
+                    self.faiss_indexes[index].train(embeddings_to_index)
```

Comment:
`embeddings_to_index` is not defined here, this would fail. Did you run this code locally?

Reply:
Sorry, wrong input......, I have updated the code here. Do I need to close this PR and open another one?

Reply:
@kaixuanliu no need for a new PR, keep pushing new commits on the existing branch `main` of your fork and this PR will be updated automatically!